platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/ConfigurationCacheIO.kt

```diff
@@ -296,7 +311,7 @@ class ConfigurationCacheIO internal constructor(
     private
     fun readerContextFor(
         inputStream: InputStream,
-    ) = readerContextFor(KryoBackedDecoder(inputStream))
+    ) = readerContextFor(KryoBackedDecoder(FramedSnappyCompressorInputStream(inputStream)))
```

Comment:
🤔 We can offload the reading as well...

Reply:
We can, however, reading seems fast enough already. Faster than without compression even.